python/src/uagents/network.py

```diff
@@ -409,28 +467,45 @@ async def register(
             agent_address (str): The address of the agent.
             name (str): The name to be registered.
             domain (str): The domain in which the name is registered.
+            overwrite (bool, optional): Specifies whether to overwrite any existing
+                addresses registered to the domain. If False, the address will be
+                appended to the previous records. Defaults to True.
         """
         logger.info("Registering name...")
         chain_id = ledger.query_chain_id()
 
-        if not get_almanac_contract(chain_id == "dorado-1").is_registered(
-            agent_address
-        ):
-            logger.warning(
-                f"Agent {name} needs to be registered in almanac contract to register its name"
-            )
-            return
+        records = parse_record_config(agent_records)
+        agent_addresses = [val.get("address") for val in records]
+
+        for agent_address in agent_addresses:
+            if not get_almanac_contract(chain_id == "dorado-1").is_registered(
+                agent_address
+            ):
+                logger.warning(
+                    f"Address {agent_address} needs to be registered in almanac contract "
+                    f"to be registered in a domain"
```

Comment:
Isn't a plus `+` needed here?
```suggestion
                    + "to be registered in a domain"
```